integrations/github.js

```diff
@@ -0,0 +1,156 @@
+module.exports = async function(robot, kredits) {
+
+  const Contributor = kredits.Contributor;
+  const Operator = kredits.Operator;
+
+  function getContributorByGithubUser(username) {
+    return Contributor.all().then(contributors => {
+      let contrib = contributors.find(c => {
+        return c.github_username === username;
+      });
+      if (!contrib) {
+        throw new Error(`No contributor found for ${username}`);A
+      } else {
+        return contrib;
+      }
+    });
+  }
+
+  function createProposal(githubUser, amount, description, url, details) {
+    return getContributorByGithubUser(githubUser).then((contributor) => {
+      robot.logger.debug(`[kredits] Creating proposal to issue ${amount}₭S to ${githubUser} for ${url}...`);
+      let contributionAttr = {
+        contributorId: contributor.id,
+        amount: amount,
+        contributorIpfsHash: contributor.ipfsHash,
+        url,
+        description,
+        details,
+        kind: 'dev'
+      };
+      return Operator.addProposal(contributionAttr).then((result) => {
+          robot.logger.debug('[kredits] proposal created:', util.inspect(result));
```

Comment:
do we need a `const util = require('util');` in this file?

Reply:
Please ignore the code until I have pushed all of my changes. Thanks.